File: linearSearch.py
#Enter the list of elements without space
    array = list(input("Enter the elements\n"))
#Enter the value of the element to be searched for
    num = input("ENter the number to be searched\n")
#Linear search:
    for i in range(len(array)):
        if(num==array[i]):
            print("Element found at position:", i)          #Prints the index of the element
            break       #If the element is not found, do nothing

File: LinearSearch.java
import java.util.*;

class LinearSearch{
    public static void main(String args[]){
        Scanner in = new Scanner(System.in);
        System.out.print("Enter the number of elements:");
        int n = in.nextInt();
        System.out.print("Enter the element value to be searched:");
        int number = in.nextInt();
        int arr1[] = new int[n];
        System.out.print("Enter the elements of the array:");
        for(int i = 0; i<n; i++) {
            arr1[i] = in.nextInt();
        }
        for(int i = 0; i<n ; i++){
            if (number==arr1[i])
                System.out.print("Element found at position:" + i);
        }
    }
}